tools/Performance/DynamoPerformanceTests/ResultsComparer.cs

```diff
@@ -105,12 +119,18 @@ internal bool BaseBenchmarkFound
                 get { return BaseResult != null; }
             }
 
+            /// <summary>
+            /// Marks the result of this comparison - should the comparison between base and new result in a test failure?
+            /// </summary>
+            internal ComparisonResultState ResultState { get; set; }
+
             /// <summary>
             /// Create a Comparison between two benchamrk results
             /// </summary>
             /// <param name="baseData"></param>
             /// <param name="newData"></param>
-            internal BenchmarkComparison(BenchmarkResult baseData, BenchmarkResult newData)
+            internal BenchmarkComparison(BenchmarkResult baseData, BenchmarkResult newData, 
+                Func<BenchmarkResult,BenchmarkResult, ComparisonResultState> comparisonFunction = null)
```

Comment:
Maybe consider default to DefaultBenchmarkComparison here instead of setting in the function body?

Reply:
sure, that makes sense - I have to see if it can be referenced here easily.